### ISSUE TITLE
can't upload logo img

### ISSUE THREAD

**Issue by vdias38** [meumobi/sitebuilder#647]
### Actual behaviour
```
[2019-03-28 14:20:52] sitebuilder.ERROR: "uploadFile failed" {"model":"[object] (SiteLogos: {\"id\":\"545\"})","path":"uploads/site_logos","message":"upload folder is not writeable","exception":"[object] (Exception(code: 0): upload folder is not writeable at /home/meumobi/PROJECTS/meumobi.com/releases/20181119114456/sitebuilder/lib/utils/FileUpload.php:87)","component":"images"}
```
### Steps to reproduce
1. login on meumobi
2. on current project try uploading logo

### Expected responses
- Why it happens
- How to fix it
- How to test

**Comment by danconte72** [comment]
## Why it happens
related to #644 
>  I've performed img- backups routine as root, the script has changed the folder mod.

```bash
meumobi@ks387594:~/PROJECTS/meumobi.com/shared/uploads$ ls -al
total 104
drwxrwsrwx 10 meumobi  meumobi  4096 Mar  7 15:08 .
drwxrwsr-x  7 meumobi  meumobi  4096 Oct  8  2014 ..
drwxrwxrwx  2     1002 meumobi  4096 Oct 16  2012 imports
drwxrwsrwx  2 root     meumobi 36864 Mar 27 21:16 items
drwxrwsr-x  2 meumobi  meumobi 36864 Mar 27 14:26 pdf_thumbnail
drwxrwsr-x  2 root     meumobi  4096 Mar 12 14:47 site_apple_touch_icon
drwxrwsr-x  2 root     meumobi  4096 Mar 12 14:47 site_logos
drwxrwsr-x  2 root     meumobi  4096 Mar 12 14:47 site_photos
drwxrwsr-x  2 root     meumobi  4096 Mar  7 15:05 site_splash_screens
```
there is no access for meumobi to pdf_thumbnail, site_apple_touch_icon, site_logos, site_photos, site_splash_screens
## How to fix
run as root
```bash
$ chmod -R 777 /shared/uploads/pdf_thumbnail
$ chmod -R 777 /shared/uploads/site_apple_touch_icon
$ chmod -R 777 /shared/uploads/site_logos
$ chmod -R 777 /shared/uploads/site_photos
$ chmod -R 777 /shared/uploads/site_splash_screens
```
A definitive fix will be set on #646

**Comment by danconte72** [comment]
I've run 
```bash
root@ks387594[ELEFANTE]:/home/meumobi/PROJECTS/meumobi.com# chmod -R 777 shared/uploads/site_logos
root@ks387594[ELEFANTE]:/home/meumobi/PROJECTS/meumobi.com# chmod -R 777 shared/uploads/pdf_thumbnail
root@ks387594[ELEFANTE]:/home/meumobi/PROJECTS/meumobi.com# chmod -R 777 shared/uploads/site_apple_touch_icon
root@ks387594[ELEFANTE]:/home/meumobi/PROJECTS/meumobi.com# chmod -R 777 shared/uploads/site_logos
root@ks387594[ELEFANTE]:/home/meumobi/PROJECTS/meumobi.com# chmod -R 777 shared/uploads/site_photos
root@ks387594[ELEFANTE]:/home/meumobi/PROJECTS/meumobi.com# chmod -R 777 shared/uploads/site_splash_screens
root@ks387594[ELEFANTE]:/home/meumobi/PROJECTS/meumobi.com# ls -al shared/uploads/
total 104
drwxrwsrwx 10 meumobi  meumobi  4096 Mar  7 15:08 .
drwxrwsr-x  7 meumobi  meumobi  4096 Oct  8  2014 ..
drwxrwxrwx  2     1002 meumobi  4096 Oct 16  2012 imports
drwxrwsrwx  2 root     meumobi 36864 Mar 27 21:16 items
drwxrwsrwx  2 meumobi  meumobi 36864 Mar 27 14:26 pdf_thumbnail
drwxrwsrwx  2 root     meumobi  4096 Mar 12 14:47 site_apple_touch_icon
drwxrwsrwx  2 root     meumobi  4096 Mar 28 15:13 site_logos
drwxrwsrwx  2 root     meumobi  4096 Mar 12 14:47 site_photos
drwxrwsrwx  2 root     meumobi  4096 Mar  7 15:05 site_splash_screens
``
`

## How to Test
1. login on meumobi
2. upload site logo
logo uploaded

**Comment by danconte72** [comment]
mod was wrongly set on backup image script, it will be fixed on #646
